src/main/java/by/exadel/internship/exception_handing/GlobalExceptionHandler.java

```diff
@@ -54,4 +61,11 @@ private IncorrectData incorrectDataFillingNotFoundException(NotFoundException ex
         incorrectData.setCodeException(exception.getCodeException());
         return incorrectData;
     }
+
+    private IncorrectData incorrectDataFillingAccessException(AccessDeniedException exception){
+        IncorrectData incorrectData = new IncorrectData();
+        log.error("Message: " + ACCESS_ERROR_MESSAGE + " Error UUID code: " + incorrectData.getErrorCode());
```

Comment:
replace concatenation with {}, review how it is done in other places

Reply:
Done